### PR TITLE
Kubecost 2.0 Install/Upgrade landing page.

### PR DESCRIPTION
**Pull request by thomasvn**
First draft of new doc `kubecostV2` which acts as a landing page for users looking for more info on what to modify before upgrading.

## Related issue

None

### CODE REVIEW

File: install-and-configure/install/kubecostv2.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+If you are using Helm, it may look something like this:
+
+```sh
+```
+
+Upon upgrading, you should see the following pods running:
+
+```sh
+```
```
Comment (bstuder99) [inline install-and-configure/install/kubecostv2.md:17]: I think something went wrong while copying and pasting content

Comment (thomasvn) [reply install-and-configure/install/kubecostv2.md:17]: Still working on the doc! Haven't added these code blocks in yet

Comment (thomasvn) [reply install-and-configure/install/kubecostv2.md]: Ok, just added them in!

**Review by bstuder99 — APPROVED** [review]

**Comment by thomasvn** [comment]
Feedback still welcome on this PR! I'll be continuously looking at docs and refining it throughout the day.

**Pull request merged**